README.md

```diff
@@ -1,25 +1,53 @@
 # Automated Test Framework (ATF) scripts
 This repository contains ATF scripts and data to run it.
 
-## Getting Started:
+[![Build Status](https://travis-ci.org/smartdevicelink/sdl_atf_test_scripts.svg?branch=master)](https://travis-ci.org/smartdevicelink/sdl_atf_test_scripts)
 
-* [Setup SDL](https://github.com/smartdevicelink/sdl_core). 
- * Later the SDL sources destination directory is referenced as `<sdl_core>`  
+## Start in Auto mode
+### Preconditions:
+For test script automatisation warkstation should be able to build SDL and ATF, also python fabric should be installed.
```

Comment:
@LuxoftAKutsan, please, double check spelling. At least "automatisation" and "warkstation" have mistakes.